pkgs/development/tools/misc/netcoredbg/default.nix

```diff
@@ -9,7 +9,7 @@ let
     owner = "dotnet";
     repo = "runtime";
     rev = coreclr-version;
-    sha256 = "sha256-kBYb0Uw1IzDTpsEyd02/5sliVHoLmZdGnpybneV0u7U=";
+    sha256 = "sha256-5BYOvOMb5cPn9AWVN9G1h1PIHhUuCh1WeeWG/QSbtzU=";
```

Comment:
So the hash was changed because we were tracking a branch, and that branch was updated, right?
I'd suggest pinning it to a tag this time, v7.0.4 came out yesterday. This should avoid future problems like that.

coreclr-version = "release/7.0"; ->   coreclr-version = "v7.0.4";

Reply:
Now there is a sha mismatch again and the app does not build (due to the switch to a tag).
I'd suggest replace the 2 pushed commits with just one that will:
- update coreclr-version to `v7.0.4`
- change its sha to `sha256-gPl9sfn3eL3AUli1gdPizDK4lciTJ1ImBcics5BA63M=`
And then we'll be ok.

Reply:
Now there is a sha mismatch again and the app does not build (due to the switch to a tag).
I'd suggest replace the 2 pushed commits with just one that will:

-    update coreclr-version to `v7.0.4`
-   change its sha to `sha256-gPl9sfn3eL3AUli1gdPizDK4lciTJ1ImBcics5BA63M=`
     And then we'll be ok.

Reply:
Now there is a sha mismatch again and the app does not build (due to the switch to a tag).
I'd suggest replace the 2 pushed commits with just one that will:

-  update coreclr-version to `v7.0.4`
-  change its sha to `sha256-gPl9sfn3eL3AUli1gdPizDK4lciTJ1ImBcics5BA63M=`

And then we'll be ok.

Reply:
Ok, i did as you suggested and hopefully now it is ok :)